rna-germline-variant-calling.wdl

```diff
@@ -0,0 +1,777 @@
+workflow RNAseq {
+
+	File inputBam
+	String sampleName = basename(inputBam,".bam")
+
+	File refFasta
+	File refFastaIndex
+	File refDict
+	
+	File wgsCallingIntervalList
+	Int? haplotypeScatterCount = 6
+	Int breakBandsAtMultiplesOf = 100000
+
+	Array[File] knownVcfs
+	Array[File] knownVcfsIndices
+
+    File dbSnpVcf
+    File dbSnpVcfIndex
+
+	## Requirements for STAR
+	Int? readLength
+	File? zippedStarReferences
+	File annotationsGTF
+  
+  	## Optional user optimizations
+    Int? maxReadsForRealignment
+    Int? compression_level
+    Int? maxRecordsForMergeBamAlignment
```

Comment:
some of these variables aren't used and should be removed

Reply:
done